lib/pagination_search/search_conditions.rb

```diff
@@ -37,8 +37,16 @@ def wrapped(cleaned_grouped_hash)
         Result.new(cleaned_grouped_hash)
       end
 
+      def quotation_safe(search_string)
+        if search_string.count('""').odd? || /\S\"\S/.match(search_string)
```

Comment:
actually i've noticed that there are issues as well when the quotes are matched ... for example `views.visit_facts` return something but `"views.visit_facts"` will not